### PR TITLE
CLOUDP-200720: Fix reported bugs

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ [CLOUDP-200720](https://jira.mongodb.org/browse/CLOUDP-200720)

1. Index creation w/ text support + index ID in output
```shell
~/workplace/mongocli CLOUDP-200720 *25 !3 ?1 ❯   atlasdev deployments search indexes create testNew  --deploymentName local5372 --db sample_mflix  --collection embedded_movies  -P oakspa                                                  1.20 17:19:42
Search index created with ID 6509c11780b2f007e6a2a0d2 
```
2. Index creation w/ json support + index ID in output
```shell
~/workplace/mongocli CLOUDP-200720 *25 !3 ?1 ❯   atlasdev deployments search indexes create testNew  --deploymentName local5372 --db sample_mflix  --collection embedded_movies                                                             1.20 17:19:51
{
  "collectionName": "embedded_movies",
  "database": "sample_mflix",
  "indexID": "6509c11780b2f007e6a2a0d2",
  "name": "testNew"
}
```

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/deployments/setup.go [hunk 1]
```diff
@@ -457,7 +457,7 @@ func (opts *SetupOpts) generateDeploymentName() {
 
 func (opts *SetupOpts) promptDeploymentName() error {
 	p := &survey.Input{
-		Message: "Cluster Name [This can't be changed later]",
+		Message: "Deployment Name [This can't be changed later]",
```
Comment (blva) [inline internal/cli/atlas/deployments/setup.go:460]: updating wording to deployment

File: internal/cli/atlas/deployments/search/indexes/create.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ import (
 const (
 	namePattern        = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
 	connectWaitSeconds = 10
-	createTemplate     = "Search index created\n"
+	createTemplate     = "Search index created with ID {{.IndexID}}\n"
```
Comment (wtrocki) [inline internal/cli/atlas/deployments/search/indexes/create.go:41]: Nit `:` or ID at the end

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: internal/cli/atlas/deployments/search/indexes/create.go (hunk 2 above)
Comment (blva) [reply internal/cli/atlas/deployments/search/indexes/create.go:41]: will add this to my next PR to avoid delays Thank you!

**Pull request merged**